simtools/util/workflow_description.py

```diff
@@ -0,0 +1,571 @@
+        self._fill_activity_meta()
+
+    def label(self):
+        """
+        Return workflow name
+        (often set as label)
+
+        """
+
+        return self.workflow_config['ACTIVITY']['NAME']
+
+    def configuration(self, key, value=None):
+        """
+        Set or Return workflow configuration parameter.
+
+        Usually filled from argparser.
+
+        """
+
+        try:
+            if value is None:
+                return self.workflow_config['CONFIGURATION'][key]
+            else:
+                self.workflow_config['CONFIGURATION'][key] = value
+        except KeyError:
+            self._logger.error("Missing key {} in CONFIGURATION".format(key))
+            raise
+
+    def reference_data_columns(self):
+        """
+        Return reference data column definition expected
+        in input data
+
+        Returns
+        -------
+        DATA_COLUMNS dict
+            reference data columns
+
+        Raises
+        ------
+        KeyError
+            if DATA_COLUMNS does not exist in workflow
+            configuration
+
+        """
+
+        try:
+            return self.workflow_config["DATA_COLUMNS"]
+        except KeyError:
+            self._logger.error(
+                "Missing DATA_COLUMNS entry in workflow configuration")
+            raise
+
+    def product_data_file_name(self, suffix=None):
+        """
+        Return full path and name of product data file
+
+        file name is determined by:
+        a. Toplevel meta ['PRODUCT']['DATA']
+        b. Toplevel meta ['PRODUCT']['ID'] + label
+
+        File name always used CTA:PRODUCT:ID for unique identification
+        (not applied when CONFIGURATION:test is true)
+
+        Returns
+        -------
+        Path
+            data file path and name
+
+        Raises
+        ------
+        KeyError
+            if data file name is not defined in workflow configuration
+            or in product metadata dict
+
+        """
+
+        _directory = self.product_data_directory()
+        try:
+            if self.workflow_config['CONFIGURATION']['test']:
+                _filename = 'TEST'
+            else:
+                _filename = self.workflow_config['ACTIVITY']['ID']
+            if self.workflow_config['PRODUCT']['FILENAME']:
+                _filename += '-' + self.workflow_config['PRODUCT']['FILENAME']
+            else:
+                _filename += '-' + self.workflow_config['ACTIVITY']['NAME']
+        except KeyError:
+            self._logger.error(
+                "Missing CTA:PRODUCT:ID in metadata")
+            raise
+
+        if not suffix:
+            suffix = '.' + self.product_data_file_format(suffix=True)
+
+        return Path(_directory).joinpath(_filename+suffix)
+
+    def product_data_file_format(self, suffix=False):
+        """
+        Return file format for data file
+
+        Parameter
+        ---------
+        suffix: bool
+            return file suffix (if true)
```

Comment:
Please change to `return just the ecsv suffix (if format is ascii.ecsv)`. The current text is a little confusing and it's not actually any suffix, it's only `.ecsv`